### PR TITLE
Feat : logout Page

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added Logout Page 

- Screenshot -
 
Screenshot from 2022-02-16 13-03-45

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
loved th two tone color scheme of the app and minimal design , good to merge

File: Pages/Authentication/logout.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          <p class="hero__sub__heading">
+            You are logges out. To see your account please log in.
+          </p>
+          <p class="hero__subheading">
+            Don't have an account? Sign in to create one
+          </p>
+
+          <div class="logout__buttons">
+            <a class="btn btn__default">Login</a>
+            <a class="btn btn--outlined outlined__secondary">Signin</a>
```
Comment (romabulani) [inline Pages/Authentication/logout.html]: Just a suggestion, here you can give signup instead of signin as login and signin have the same meaning

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

File: Pages/Authentication/logout.html (hunk 1 above)
Comment (anuj-kumary) [reply Pages/Authentication/logout.html]: Ohh thanks for pointing out

File: Pages/Authentication/logout.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <body>
+    <main class="logout__container">
+      <section class="logout">
+        <div class="logout__content">
+          <h3 class="hero__heading">You are logged out</h3>
+          <p class="hero__sub__heading">
+            You are logges out. To see your account please log in.
+          </p>
+          <p class="hero__subheading">
+            Don't have an account? Sign in to create one
```
Comment (rahulbakshi4) [inline Pages/Authentication/logout.html]: hey just a suggestion, here it should be Sign Up to create one.

**Review by rahulbakshi4 — APPROVED** [review]
Rest all looks good !!

**Pull request merged**